tests/io/test_particle_collection.py

```diff
@@ -23,7 +23,7 @@
     pid=443,
     mass=3.0969,
     width=9.29e-05,
-    state=ParticleQuantumState(
+    state=QuantumState[float](
```

Comment:
Is there a way to provide a 'default' template argument (and, if so, would this be a desired feature)?

Reply:
Not that I know. In my opinion a default would also not bring much benefit. You can make a "type alias" within a class to avoid retyping the same class type `_ConcreteQuantumState = QuantumState[float]`